cockpit/gui/mainWindow.py

```diff
@@ -94,6 +96,14 @@ class MainWindow(wx.Frame):
     def __init__(self):
         wx.Frame.__init__(self, parent = None, title = "Cockpit program")
 
+        #set window icon
+        self.bitmapsPath = cockpit.gui.BITMAPS_PATH
+        icon = wx.Icon(wx.Bitmap(os.path.join( self.bitmapsPath,
+                                               "cockpit.ico"), 
+                                 wx.BITMAP_TYPE_ANY))
```

Comment:
From the documentation, it seems that you can just pass the path to `wx.Icon`, no need to create the `wx.Bitmap` explicitely. Also, we know we are a reading an ico file so either pass `wx.BITMAP_TYPE_ICO` if required, or pass nothing since `ANY` is already the default.